src/commands/mount.rs

```diff
@@ -78,24 +97,32 @@ impl Runnable for MountCmd {
 }
 
 impl MountCmd {
+    fn new() -> Self {
```

Comment:
I would prefer a different name: `default_config()`. As this is not `pub` currently, we are actually totally free to choose any name ;-)